02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,37 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Clint Lucio Bio</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
-
+     <body style="text-align:center;">
+       <div>
+     </body>
+     <img src="http://feelgrafix.com/data_images/out/28/979988-bio.jpg"alt="Mountain View" style="width:304px;height:228px;">
+              <h1>Clint Lucio</h1>
+                <p>Austin,Texas</p>
+        <h2>Bio</h2>
+            <p>I was born in Austin, the oldest of 4 children.</p>
+            <p> I started working in the Semiconductor Industry in 1997.</p>
+            <p>For a large portion of my career, I've been working for Samsung Austin Semiconductor.</p>
+              <h2>Hobbies.</h2>
+                 <p>Building computers</p>
+                 <p>Restoring cars.</p>
+                   <img src="https://scontent.xx.fbcdn.net/v/t1.0-9/38112_420120167491_3828379_n.jpg?oh=7ee97332caad8a1fc3a6d138500c0143&oe=581BF730" style="width:400px;height:300px"/>
```

Comment:
nesting and formatting can be better. try installing and using the atom-beautify package.